bgcval2/bgcvaltools/makeMask.py

```diff
@@ -216,20 +216,20 @@ def makeMask(name, newSlice, xt, xz, xy, xx, xd, debug=False):
     if newSlice == 'ignoreExtraArtics':
         return np.ma.masked_outside(xy, -50., 50.).mask
     if newSlice == 'NorthAtlanticOcean':
-        return np.ma.masked_outside(bvt.makeLonSafeArr(xx), -80.,
+        print('NorthAtlanticOcean - pre')
+        #xx = bvt.makeLonSafeArr(xx)
+        print('NorthAtlanticOcean', len(xx))
```

Comment:
```suggestion
```

Reply:
yo can you maybe not do development in review mode, pls bud? I get an email everytime you "suggest" a change :rofl:

Reply:
Github is annoying like that. I was told there's a setting somewhere that reduces the number of emails. I don't believe it personally.